metadata/site_import/services.xml

```diff
@@ -34,13 +34,18 @@
         <url>https://terminal-api-test.adyen.com/sync</url>
         <user-id></user-id>
         <password/>
-    </service-credential>   
+    </service-credential>
     <service-credential service-credential-id="AdyenOriginKeys">
         <url>https://checkout-test.adyen.com/v1/originKeys</url>
         <user-id></user-id>
         <password/>
-    </service-credential>   
-     
+    </service-credential>
+    <service-credential service-credential-id="AdyenCheckoutPaymentMethods">
+        <url>https://checkout-test.adyen.com/v40/paymentMethods</url>
+        <user-id></user-id>
+        <password/>
```

Comment:
any possibilities for x-api-key ?

Reply:
API key is taken from Custom Preferences, so username/password is not required there.